test/onigumo_test.exs

```diff
@@ -9,6 +9,22 @@ defmodule OnigumoTest do
 
   setup(:verify_on_exit!)
 
+  @tag :tmp_dir
+  test("run the download", %{tmp_dir: tmp_dir}) do
+    expect(HTTPoisonMock, :start, fn -> nil end)
+    expect(HTTPoisonMock, :get!, length(@urls), &prepare_response/1)
+
+    input_path_env = Application.get_env(:onigumo, :input_path)
+    input_path_tmp = Path.join(tmp_dir, input_path_env)
+    input_file_content = prepare_input(@urls)
+    File.write!(input_path_tmp, input_file_content)
+
+    stream_result = Onigumo.Downloader.main(tmp_dir)
+    assert(stream_result == :ok)
```

Comment:
This assertion is redundant and lies here for consistency with [a result](https://github.com/Glutexo/onigumo/blob/5d0a6a4f81ab69ccdc2c7df0d74ce89a85d36e99/test/onigumo_test.exs#L34) of _File.write!_ in another test.

Reply:
I am not sure what do you want to say....
I suggest the :ok is return value from `Stream.run()` when I've read the docs - https://hexdocs.pm/elixir/Stream.html#run/1